source/code-of-conduct/index.html.haml

```diff
@@ -1,44 +1,39 @@
 ---
-title: Brug - Code of Conduct - Ruby Belgium
+title: Code of Conduct - Ruby Belgium
 ---
 
-%header.header
+%header
```

Comment:
Imho html represent the structure and css how it should be styled. Using html tag is a general bad practice because it's not context dependent. 
If you mind the naming redundancy we could make it more explicit with ```%header.page-heading``` and also be more consistent with section--heading at the same time